File: HELLO_WEB/src/main/java/kr/co/aiai/dao/MyUpdate.java
package kr.co.aiai.dao;
import java.sql.*;

public class MyUpdate {
	
	public static void main(String[] args) throws Exception {

		Class.forName("com.mysql.jdbc.Driver");
		Connection conn = DriverManager.getConnection("jdbc:mysql://127.0.0.1:3305/python",
													"root", "python");

		String sql = " update emp" + 
					" set e_name=?, sex=?, addr=? " + 
					" where e_id = ? ";
		PreparedStatement pstmt = conn.prepareStatement(sql);
		pstmt.setString(1, "4");
		pstmt.setString(2, "4");
		pstmt.setString(3, "4");
		pstmt.setInt(4, 3);
		

		int cnt = pstmt.executeUpdate();
		
		System.out.println(cnt + "개 행 update 성공");
		
		pstmt.close();
		conn.close();

	}
}

File: day07/myupdate.py
import pymysql
conn = pymysql.connect(host='127.0.0.1',
                       port=3305,
                       user='root',
                       password='python',
                       db='python',
                       charset='utf8'
                       )

cursor = conn.cursor()
e_id = '3'
e_name = '5'
sex = '4'
addr = '4'

# f string은 python 3.5 이상부터 사용가능...
sql = f"""
    update emp
    set
        e_name = '{e_name}',
        sex = '{sex}',
        addr = '{addr}'
    where
        e_id = '{e_id}'
"""

cnt = cursor.execute(sql)
print(cnt)

conn.commit()
cursor.close()
conn.close()